diff --git a/net/wireless/reg.c b/net/wireless/reg.c
--- a/net/wireless/reg.c
+++ b/net/wireless/reg.c
@@ -233,7 +233,7 @@ static const struct ieee80211_regdomain world_regdom = {
 		REG_RULE(4920-10, 5240+10, 160, 6, 30, 0),
 
 		/* IEEE 802.11a, channel 52..64 - DFS required */
-		REG_RULE(5260-10, 5320+10, 80, 6, 30, 0),
+		REG_RULE(5260-10, 5500+10, 80, 6, 30, 0),
 
 		/* IEEE 802.11a, channel 100..144 - DFS required */
 		REG_RULE(5500-10, 5720+10, 160, 6, 30, 0),
